src/Tools/FileIssues/Program.cs

```diff
@@ -184,6 +241,15 @@ private NewIssue CreateNewIssue(PortingInfo ruleToPort, string title)
                 case Disposition.Cut:
                     AddLabel(CutLabel, newIssue.Labels);
                     break;
+
+                case Disposition.Ported:
+                    AddLabel(PortedLabel, newIssue.Labels);
+                    break;
+            }
+
+            if (ruleToPort.Soon)
+            {
+                AddLabel(UrgencySoonLabel, newIssue.Labels);
```

Comment:
This is a new issue; we don't have to worry about removing the label. But see the Update code below.